src/filters/simple-filter-directive.js

```diff
@@ -0,0 +1,280 @@
+            address: "2 Apple Boulevard, Cincinatti, Ohio",
+            birthMonth: 'December'
+          },
+          {
+            name: "Pat Thomas",
+            address: "50 Second Street, New York, New York",
+            birthMonth: 'February'
+          }
+        ];
+        $scope.items = $scope.allItems;
+
+        var matchesFilter = function (item, filter) {
+          var match = true;
+
+          if (filter.id === 'name') {
+            match = item.name.match(filter.value) !== null;
+          } else if (filter.id === 'address') {
+            match = item.address.match(filter.value) !== null;
+          } else if (filter.id === 'birthMonth') {
+            match = item.birthMonth === filter.value;
+          }
+          return match;
+        };
+
+        var matchesFilters = function (item, filters) {
+          var matches = true;
+
+          filters.forEach(function(filter) {
+            if (!matchesFilter(item, filter)) {
+              matches = false;
+              return false;
+            }
+          });
+          return matches;
+        };
+
+        var applyFilters = function (filters) {
+          $scope.items = [];
+          if (filters && filters.length > 0) {
+            $scope.allItems.forEach(function (item) {
+              if (matchesFilters(item, filters)) {
+                $scope.items.push(item);
+              }
+            });
+          } else {
+            $scope.items = $scope.allItems;
+          }
+          $scope.filterConfig.resultsCount = $scope.items.length;
+        };
+
+        var filterChange = function (filters) {
+        $scope.filtersText = "";
+          filters.forEach(function (filter) {
+            $scope.filtersText += filter.title + " : " + filter.value + "\n";
+          });
+          applyFilters(filters);
+        };
+
+        $scope.filterConfig = {
+          fields: [
+            {
+              id: 'name',
+              title:  'Name',
+              placeholder: 'Filter by Name',
+              filterType: 'text'
+            },
+            {
+              id: 'address',
+              title:  'Address',
+              placeholder: 'Filter by Address',
+              filterType: 'text'
+            },
+            {
+              id: 'birthMonth',
+              title:  'Birth Month',
+              placeholder: 'Filter by Birth Month',
+              filterType: 'select',
+              filterValues: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
+            }
+          ],
+          resultsCount: $scope.items.length,
+          appliedFilters: [],
+          onFilterChange: filterChange
+        };
+      }
+    ]);
+  </file>
+</example>
+ */
+angular.module('patternfly.filters').directive('pfSimpleFilter',
+  function ($document) {
+    'use strict';
+    return {
+      restrict: 'A',
+      scope: {
+        config: '='
+      },
+      replace: true,
```

Comment:
Are we sure we want to use this [deprecated option](https://docs.angularjs.org/api/ng/service/$compile#-replace-deprecated-will-be-removed-in-next-major-release-i-e-v2-0-)?